### PR TITLE
docs: added requested text entry copy

### LINKED ISSUE #1477: Content: Add characters remaining guidance to text input
## Overview

Under /components/form/usage we include a screenshot containing characters remaining on text input, but no guidance for using this feature. We had a user ask about guidance, and we should provide it.

### Screenshot on page:

Screen Shot 2019-04-10 at 10 57 53 AM

### Slack request:

Screen Shot 2019-04-10 at 10 41 05 AM

### PR DESCRIPTION
**Pull request by connor-leech**
Closes #1477 

Added requested text entry copy. Also restructured page.

### CODE REVIEW

File: src/content/components/text-input/usage.mdx [hunk 1]
```diff
@@ -15,11 +30,39 @@ _Text inputs_ enable the user to interact with and input data. Use when the appl
...
+
+Text fields and text areas may have a character limit and counter. Use the character limit and counter to let users know how long their entry can be. 
+
+<ImageComponent cols="8" caption="An example of a text area.">
+
+![An example of a text area.](images/text-area-usage-1.png)
+
+</ImageComponent>
+
+While useful for data collection, character count is not an intuitive way to communicate content length to users. Character count allows a user to clearly understand an input error if they surpass the character limit.
```
Comment (shixiedesign) [inline src/content/components/text-input/usage.mdx]: I think it's better to keep all this content above the image, so it's less broken up.

File: src/content/components/text-input/usage.mdx [hunk 2]
```diff
@@ -15,11 +30,39 @@ _Text inputs_ enable the user to interact with and input data. Use when the appl
...
+
+<ImageComponent cols="8" caption="Error example">
+
+![Validation text example](images/text-input-usage-2.png)
+
+</ImageComponent>
+
+### Character count
+
+Text fields and text areas may have a character limit and counter. Use the character limit and counter to let users know how long their entry can be.
```
Comment (shixiedesign) [inline src/content/components/text-input/usage.mdx]: > to let users know how long their entry can be **a head of time** 

I think the "a head of time" is important to keep.

Comment (connor-leech) [reply src/content/components/text-input/usage.mdx]: It's in!

File: src/content/components/text-input/usage.mdx (hunk 1 above)
Comment (connor-leech) [reply src/content/components/text-input/usage.mdx]: Sure thing. Changed!

**Pull request merged**